CONTRIBUTING.md

```diff
@@ -3,18 +3,16 @@
 Building Cabal for hacking
 --------------------------
 
-The current recommended way of developing Cabal is to use the
-`v2-build` feature which [shipped in cabal-install-1.24](http://blog.ezyang.com/2016/05/announcing-cabal-new-build-nix-style-local-builds/).
 If you use the latest version of cabal published on Hackage, it is sufficient to run:
 
 ```
-cabal v2-build cabal
+cabal build cabal
 ```
 
 If not, you aren't able to build the testsuite, so you need to disable the default `cabal.project` that implies configuring the testsuite, e.g., with:
 
 ```
-cabal v2-build --project-file=cabal.project.release cabal
+cabal build --project-file=cabal.project.release cabal
```

Comment:
Same here. (Compare the next diff block.)

Reply:
Actually, looking at the diff below, I think lowercase is correct here. Lowercase is building the entire project, where as the below variation is building individual components. Note how there's examples for both `Cabal-tests` and `cabal-tests`.